pages/documentation/developer/repositories/pypowsybl.md

```diff
@@ -26,7 +32,8 @@ The available features are:
 
 | Version | Release date | Release notes | API documentation |
 | ------- | ------------ | ------------- | ----------------- |
-| 0.11.0 | 2021-11-22 | [Release notes](https://github.com/powsybl/pypowsybl/releases/tag/v0.11.0) | [readthedocs.io](https://pypowsybl.readthedocs.io/en/latest/) |
+| 0.12.0 | 2022-01-19 | [Release notes](https://github.com/powsybl/pypowsybl/releases/tag/v0.12.0) | [readthedocs.io](https://pypowsybl.readthedocs.io/en/latest/) |
```

Comment:
Better to point to https://pypowsybl.readthedocs.io/en/stable/ actually (which I did not do in the release annoucement :) )